src/predict/requirements.txt

```diff
@@ -38,42 +44,55 @@ gunicorn==20.0.4
 h5py==2.10.0
 idna==2.10
 importlib-metadata==1.7.0
+iniconfig==1.1.1
 itsdangerous==1.1.0
 Jinja2==2.11.2
 joblib==0.16.0
 Keras-Preprocessing==1.1.2
 kiwisolver==1.2.0
+libcst==0.3.18
 lightgbm==2.3.1
 Markdown==3.3.3
 MarkupSafe==1.1.1
 marshmallow==3.2.2
 matplotlib==3.1.1
+mongomock==3.22.1
 more-itertools==8.4.0
 multipledispatch==0.6.0
+munch==2.5.0
+mypy-extensions==0.4.3
 numpy==1.17.4
 oauthlib==3.1.0
 opt-einsum==3.3.0
 packaging==20.4
 pandas==0.25.3
 patsy==0.5.1
 pluggy==0.13.1
+proto-plus==1.18.1
 protobuf==3.12.2
 py==1.9.0
+pyarrow==3.0.0
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
 pycparser==2.20
 pymongo==3.10.1
 pyparsing==2.4.7
-pytest==4.6.11
+pyproj==3.1.0
```

Comment:
I guess this is the requirement I failed to install. Could it be the version number? Or could it be something wrong with my DEV environment?

On a side note, the requirements seem to be many, are all of them being utilised? If yes, never mind.... :)

Reply:
There is nothing wrong with the version as it exists but I think a Dockerfile will e the best option to run this once it is its own microservice